batch/batch/driver/templates/pool.html

```diff
@@ -115,6 +115,15 @@ <h2>Instances</h2>
     <td class="numeric-cell">{{ instance.failed_request_count }}</td>
     <td>{{ instance.time_created_str() }}</td>
     <td>{{ instance.last_updated_str() }} ago</td>
+    <td>
+      <form action="{{ base_path }}/instances/{{ instance.name }}/deactivate" method="post">
+        <input type="hidden" name="_csrf" value="{{ csrf_token }}" />
+        {% if q is not none %}
+        <input type="hidden" name="q" value="{{ q }}" />
+        {% endif %}
+        <button>Deactivate</button>
```

Comment:
This is a bit misleading now since deactivate means something very specific in our application and this is calling `instance.kill`. Delete is a less ambiguous synonym for `kill` that is also a natural button name.